### PR TITLE
fix: text link transition

### PR DESCRIPTION
**Pull request by vikashchander**
hi, 

try to fix **Import page link text animates at a different speed compared to the rest of the page's text #65**

### CODE REVIEW

**Comment by kartikcho** [comment]
Is there a way to keep the hover transitions and fixing the text resize speed without removing them at all? Can you look into it if possible @vikashchander ?

**Comment by vikashchander** [comment]
> Is there a way to keep the hover transitions and fixing the text resize speed without removing them at all? Can you look into it if possible @vikashchander ?

yeah, I can try through changing value of transitions

**Comment by kartikcho** [comment]
> 
> yeah, I can try through changing value of transitions

That would still cause the issue to persist, just faster. Maybe there's a specific way around this you might want to look into.

**Comment by vikashchander** [comment]
> > yeah, I can try through changing value of transitions
> 
> That would still cause the issue to persist, just faster. Maybe there's a specific way around this you might want to look into.

could you suggest me any specific way ??

File: src/ImportPage.css [hunk 1]
```diff
@@ -14,11 +14,9 @@
 
 .link {
   color: #282c34;
-  transition: 0.2s;
```
Comment (kartikcho) [inline src/ImportPage.css:17]: Keeping this in seems to fix the issue without removing the animations. Can you try and confirm?

**Review by kartikcho — CHANGES_REQUESTED** [review]
Good work, checkout my comment and let's get this merged!

File: src/ImportPage.css (hunk 1 above)
Comment (vikashchander) [reply src/ImportPage.css:17]: yeah I can try

**Comment by kartikcho** [comment]
Looks good, could you rename your branch to `text-animation-fix`. Also as a future note, don't name your branch as your name.
Let's get this merged soon!

**Pull request closed without merging**